src/main/scala/org/allenai/deep_qa/experiments/Models.scala

```diff
@@ -46,10 +46,94 @@ object Models {
     ("num_memory_layers" -> numMemoryLayers)
 
   val attentionSumReader: JValue =
-    ("model_class" -> "AttentionSumReader")
+    ("model_class" -> "AttentionSumReader") ~
+    ("patience" -> 1) ~
+    ("preferred_backend" -> "theano") ~
+    ("encoder" ->
+      ("default" ->
+        ("type" -> "bi_gru")~
+        ("output_dim" -> 384)
+      )
+    ) ~
+    ("seq2seq_encoder" ->
+      ("default" ->
+        ("type" -> "bi_gru") ~
+        ("encoder_params" ->
+          ("output_dim" -> 384)
+        ) ~
+        ("wrapper_params" -> JObject())
+      )
+    ) ~
+    ("optimizer" ->
+      ("type" -> "adam") ~
+      ("clipnorm" -> 10.0) ~
+      ("lr" -> 0.0005)
+    ) ~
+    ("embedding_dropout" -> 0.0) ~
+    ("patience" -> 0) ~
+    ("embedding_size" -> 256) ~
+    ("num_epochs" -> 5)
 
   val gatedAttentionReader: JValue =
-    ("model_class" -> "GatedAttentionReader")
+    ("model_class" -> "GatedAttentionReader") ~
+    ("embedding_size" -> 100) ~
+    ("pretrained_embeddings_file" -> "/efs/data/dlfa/glove/glove.6B.100d.txt.gz") ~
+    ("fine_tune_embeddings" -> false) ~
+    ("project_embeddings" -> false) ~
+    ("num_gated_attention_layers" -> 3) ~
+    ("patience" -> 0) ~
+    ("preferred_backend" -> "theano") ~
+    ("encoder" ->
+      ("question_final" ->
+        ("type" -> "bi_gru")~
+        ("output_dim" -> 128)
+      )
+    ) ~
+    ("seq2seq_encoder" ->
+      ("question_0" ->
```

Comment:
This looks fine; feel free to merge.  The only thing you might think about changing is using the `default` key here, so you don't have to repeat yourself with all of these encoders.  Note that this requires setting the `fallback_behavior` in [these lines](https://github.com/allenai/deep_qa/pull/187/files#diff-58cd40374724399ec279f21d52c0bec5R109) (which I think you should do, anyway).